app/tests/components_tests/test_views.py

```diff
@@ -539,3 +544,103 @@ def test_interfaces_list_link_in_new_interface_form(
         user=u,
     )
     assert reverse(interface_list_viewname) in response.rendered_content
+
+
+@pytest.mark.parametrize(
+    "base_object_factory,base_obj_lookup,object_factory,viewname,is_edit_view",
+    (
+        (
+            ReaderStudyFactory,
+            "reader_study",
+            DisplaySetFactory,
+            "reader-studies:display-set-create",
+            False,
+        ),
+        (
+            ReaderStudyFactory,
+            "reader_study",
+            DisplaySetFactory,
+            "reader-studies:display-set-update",
+            True,
+        ),
+        (
+            ArchiveFactory,
+            "archive",
+            ArchiveItemFactory,
+            "archives:item-create",
+            False,
+        ),
+        (
+            ArchiveFactory,
+            "archive",
+            ArchiveItemFactory,
+            "archives:item-edit",
+            True,
+        ),
+    ),
+)
+@pytest.mark.parametrize(
+    "widget_type_object_factory",
+    (
+        ImageFactory,
+        UserUploadFactory,
+    ),
+)
+@pytest.mark.django_db
+def test_image_widget_populated_value_on_form_view_validation_error(
+    client,
+    base_object_factory,
+    base_obj_lookup,
+    object_factory,
+    viewname,
+    is_edit_view,
+    widget_type_object_factory,
+):
+    image = ImageFactory()
+    image_ci = ComponentInterfaceFactory(kind=ComponentInterface.Kind.IMAGE)
+    image_civ = ComponentInterfaceValueFactory(interface=image_ci, image=image)
+
+    annotation = "{}"
+    annotation_ci = ComponentInterfaceFactory(
+        kind=ComponentInterface.Kind.TWO_D_BOUNDING_BOX
+    )
+    annotation_civ = ComponentInterfaceValueFactory(
+        interface=annotation_ci, value=annotation
+    )
+
+    editor = UserFactory()
+    base_obj = base_object_factory()
+    base_obj.add_editor(editor)
+
+    object = object_factory(**{base_obj_lookup: base_obj})
+    object.values.set([image_civ, annotation_civ])
+
+    wt_object = widget_type_object_factory()
+    data = {
+        "interface_slug": image_ci.slug,
+        "current_value": wt_object.pk,
```

Comment:
These two are unnecessary for what you're testing here, I think

Reply:
I will test it and remove them if not necessary.

Reply:
outdated